pkg/util/azureclient/mgmt/network/virtualnetworks_addons.go

```diff
@@ -0,0 +1,33 @@
+package network
```

Comment:
Damn. I added this one back on a rebase. We don't need this file (I removed it in pkg/util/azureclient/mgmt/network/virtualnetworks_addons.go).

Reply:
Removed